src/mono/netcore/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -107,9 +107,13 @@ internal static bool NotifyWorkItemComplete()
         }
 
         [DynamicDependency("Callback")]
+        [DynamicDependency("PumpThreadPool")]
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         private static extern void QueueCallback();
 
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        private static extern void PumpThreadPool(); // NOTE: this method is called via reflection by test code
```

Comment:
If it's not called from managed, does this need to be added to the LinkerDescriptor?

Reply:
It's covered by DynamicDependency above